MODULE.bazel

```diff
@@ -0,0 +1,72 @@
+module(
+    name = "rules_kotlin",
+    compatibility_level = 1,
+    repo_name = "dev_io_bazel_rules_kotlin",
+    version = "1.6.0",
+)
+
+bazel_dep(name = "platforms", version = "0.0.4")
+bazel_dep(name = "rules_pkg", version = "0.7.0")
+bazel_dep(name = "bazel_skylib", version = "1.2.1")
+bazel_dep(name = "stardoc", repo_name = "io_bazel_stardoc", version = "0.5.3")
+bazel_dep(name = "rules_java", version = "5.1.0")
+
+kt = use_extension("//src/main/starlark/core/repositories:bzlmod_setup.bzl", "kt_bzlmod_ext")
+
+use_repo(kt, "rkt_1_7")
+
+use_repo(kt, "rkt_1_6")
+
+use_repo(kt, "rkt_1_5")
+
+use_repo(kt, "rkt_1_4")
+
+use_repo(kt, "legacy")
+
+use_repo(kt, "com_github_jetbrains_kotlin")
+
+use_repo(kt, "kt_java_stub_template")
+
+use_repo(kt, "com_github_pinterest_ktlint")
+
+bazel_dep(name = "rules_proto", version = "4.0.0")
+bazel_dep(name = "rules_android", repo_name = "build_bazel_rules_android", version = "0.1.1")
+bazel_dep(name = "rules_jvm_external", version = "4.4.2")
+
+maven = use_extension("@rules_jvm_external//:extensions.bzl", "maven")
+
+maven.install(
+    name = "kotlin_rules_maven",
+    artifacts = [
+        "com.google.code.findbugs:jsr305:3.0.2",
+        "junit:junit:4.13-beta-3",
+        "com.google.protobuf:protobuf-java:3.6.0",
+        "com.google.protobuf:protobuf-java-util:3.6.0",
+        "com.google.guava:guava:27.1-jre",
+        "com.google.truth:truth:0.45",
+        "com.google.auto.service:auto-service:1.0-rc5",
+        "com.google.auto.service:auto-service-annotations:1.0-rc5",
+        "com.google.auto.value:auto-value:1.6.5",
+        "com.google.auto.value:auto-value-annotations:1.6.5",
+        "com.google.dagger:dagger:2.43.2",
+        "com.google.dagger:dagger-compiler:2.43.2",
+        "com.google.dagger:dagger-producers:2.43.2",
+        "javax.annotation:javax.annotation-api:1.3.2",
+        "javax.inject:javax.inject:1",
+        "org.pantsbuild:jarjar:1.7.2",
+        "org.jetbrains.kotlinx:atomicfu-js:0.15.2",
+        "org.jetbrains.kotlinx:kotlinx-serialization-runtime:1.0-M1-1.4.0-rc",
+    ],
+    fetch_sources = True,
+    repositories = [
+        "https://maven-central.storage.googleapis.com/repos/central/data/",
+        "https://maven.google.com",
+        "https://repo1.maven.org/maven2",
+    ],
+)
+
+use_repo(maven, "kotlin_rules_maven")
+
+bazel_dep(name = "rules_python", version = "0.12.0")
```

Comment:
is it possible to mark this with `dev_dependency = True`?

Reply:
No, because on `load("@rules_python//python:defs.bzl", "py_binary")` is used in release.